ci/test.bat

```diff
@@ -68,7 +67,7 @@ exit /b 0
     set cmake_dir=cmake-build-%arcdir%-%vs_version%-%build_type%
     call %utils_script% :get_zip_file_name %component_name% %component_version%
     call %build_script% :get_version
-    if not defined GITHUB_ACTIONS (
+    if defined JENKINS_URL (
         call %utils_script% :download_from_sfc_jenkins %platform% %build_type% %vs_version% %component_name% %version%
         if !ERRORLEVEL! NEQ 0 goto :error
         dir artifacts
```

Comment:
Download cmake artifacts only for Jenkins.
For local dev, you should most likely build it.